tests/sandbox/sandbox.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+set -e
+
+PATH_HERE=$(dirname "$(realpath -s "$0")")
+VERSION_FILE="${PATH_HERE}"/../share/VERSION
+GOV_SCRIPT="${PATH_HERE}"/sandbox_gov.lua
+
+echo "Setting up Python environment..."
+
+if [ ! -f "sandbox_env/bin/activate" ]; then
+    python3.8 -m venv sandbox_env
+fi
+source sandbox_env/bin/activate
```

Comment:
Can we add `ccf_` to this name? Just thinking about the actual user flow of running this from a CCF install, if we're going to add anything to the local directory we should name it clearly. I don't know how common this impromptu venv creation is - should this be under `/tmp`?

Reply:
I changed it to `.venv_ccf_sandbox` (the [Python docs](https://docs.python.org/3/tutorial/venv.html#creating-virtual-environments) say that `.venv` is a common name) and allow overriding it by setting the `VENV_DIR` environment variable, if someone really wants to.